File: c codes/armstrong.c
#include<stdio.h>
void main()
{
    int n,t,sum=0,rem;

    printf("The armstrong numbers from 1 to 100 are: ");

    for(n=1;n<=100;n++)
    {
    t=n;
    while(t!=0)
    {
        rem=t%10;
        sum=sum+rem*rem*rem;
        t=t/10;
    }

    if(sum==n)
    {
        printf("%d",n);
    }
    }
}

File: java codes/armstrong.java
class F
{
	public static void main(String args[])
	{
		int a=153;
		int r=0;
		int sum=0;
		int num=a;
		while(a!=0)
		{
			r=a%10;
			a=a/10;
			sum=sum+(r*r*r);
		}
		if(sum==num)
		{
			System.out.println("Amstrong number");
		}
		else
		{
			System.out.println("Not a armstrong number");
		}
		

	}
}